src/Workspaces/Core/Portable/Workspace/Host/DocumentService/IDocumentOperationService.cs

```diff
@@ -1,22 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-namespace Microsoft.CodeAnalysis.Host
-{
-    /// <summary>
-    /// provide various operations for this document
-    /// 
-    /// I followed name from EditorOperation for now. 
-    /// </summary>
-    internal interface IDocumentOperationService : IDocumentService
```

Comment:
This interface is implemented in Razor via IVT, so this will break:

https://github.com/aspnet/AspNetCore-Tooling/blob/17a95db34fdfb3afa04ac42a37b68183f667f710/src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/RazorDocumentServiceProvider.cs#L10

Reply:
I see. Weird, I tested the ASP.NET Core app. I guess I'll have to add a new service then.

Reply:
Fixed.